docs/theme/da_theme_skeleton/navbar.html

```diff
@@ -2,7 +2,7 @@
 <!-- SPDX-License-Identifier: Apache-2.0 -->
 
 <div class="navbar">
-    <h1><img src="{{ pathto('_static/images/DAML_Logo_Blue.svg', 1) }}"></h1>
+    <h1><img src="{{ pathto('_static/images/Daml_Logo_Blue.svg', 1) }}"></h1>
```

Comment:
Was this file renamed appropriately?

Reply:
🤦 

Yes, it was, on my local machine. On a case-insensitive filesystem. I'm so not used to that. 😿